i18n/locales/source/dashboard/base.yml

```diff
@@ -41,10 +41,11 @@ en:
     attributes:
       user:
         name: 'Display Name'
-        name_example: 'Display Name<br/>(e.g. Cool Coder or Jane S.)'
+        name_example: 'Display Name(e.g. Cool Coder or Jane S.)'
```

Comment:
@daynew have you confirmed that this change doesn't result in a visual change on the signup page?